packages/gatsby/src/joi-schemas/joi.ts

```diff
@@ -49,6 +49,7 @@ export const gatsbyConfigSchema: Joi.ObjectSchema<IGatsbyConfig> = Joi.object()
       )
       .single(),
     developMiddleware: Joi.func(),
+    jsxAutomaticRuntime: Joi.boolean().default(false),
```

Comment:
Could we move to these 2 options please.
```suggestion
    jsxRuntime: Joi.string().default('classic'),
    jsxImportSource: Joi.string()
```